test/browser/run-test.sh

```diff
@@ -7,15 +7,9 @@ git init
 rm -f bots  # common local case: existing bots symlink
 make bots test/common
 
-# support running from clean git tree
-if [ ! -d node_modules/chrome-remote-interface ]; then
-    # copy package.json temporarily otherwise npm might try to install the dependencies from it
-    rm -f package-lock.json  # otherwise the command below installs *everything*, argh
-    mv package.json .package.json
-    # only install a subset to save time/space
-    npm install chrome-remote-interface sizzle
-    mv .package.json package.json
-fi
+# ensure current node_modules for testing
+rm -rf node_modules
+tools/node-modules checkout
```

Comment:
Still weird that there is a node_modules, but this looks much better then the hacky workaround code we had.

Reply:
Right, I just gave up trying to understand it, sorry. I have not the slightest idea where it's coming from :cry: